client/components/ProductOverview/StyleSelector.jsx

```diff
@@ -16,9 +16,13 @@ const StyleSelector = ({ styles, currentStyle, updateCurrentStyle }) => {
         <div className="stylesAvailable">
           {styles.map((style, index) => (
             <span className="stylesAvailable__list-item" key={style.style_id}>
-              <input
-                type="image"
-                src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/89/HD_transparent_picture.png/1200px-HD_transparent_picture.png"
+              <div
+                role="button"
+                onKeyPress={(e) => {
```

Comment:
This works for now, but ideally it should be in a handler

Reply:
created handler